v-next/hardhat-utils/src/request.ts

```diff
@@ -255,6 +280,17 @@ export async function getDispatcher(
   }
 }
 
+export async function getTestDispatcher(
```

Comment:
Shouldn't this be in the helpers instead? I find it confusing that's defined in a test and reexported form a helper.